### ISSUE TITLE
jquery "change" event is not fired on autonumeric control

### ISSUE THREAD

**Issue by VoimiX** [autoNumeric/autoNumeric#520]
after binding jquery event and changing a text input
```
 $(document).on("change",  "#id1", function() {
 	alert('change fired');
 });
```

the event is not fired

According to documentation

> AutoNumeric elements are transparent to the native input and change events, which means those are correctly sent when using an <input> element managed by AutoNumeric.

browsers  - all (ie, ff, chrome)
autonumeric version - 4.0.3

live example - https://jsfiddle.net/qm01xh6v/40/

**Comment by AlexandreBonneau** [comment]
This is due to jQuery handling events in its own *(weird)* manner.

You can see in that [codepen](https://codepen.io/AnotherLinuxUser/pen/BweJOv?editors=1011) that the `change` event is sent correctly and you can catch it using the default HTML5 functions.

You can check the history behind this 'issue' on https://github.com/autoNumeric/autoNumeric/issues/292, [other comments](https://github.com/autoNumeric/autoNumeric/issues/247#issuecomment-260171632) and [issues](https://github.com/autoNumeric/autoNumeric/issues/217).

So to solve your problem, I'd recommend [not using jQuery](http://youmightnotneedjquery.com/)!

**Comment by bindrid** [comment]
So, when I do the recommended action above, the change event fires twice.  How do I tell which event is fired because of the user actions and the event that was caused  by AutoNumeric. I don't want to run the callback twice with the same information.

**Comment by AlexandreBonneau** [comment]
This sounds like a bug, the `change` event is not supposed to be sent twice.
Could you create an issue with a reproducible test case?

**Comment by bindrid** [comment]
It fires twice even in your example provide above https://codepen.io/AnotherLinuxUser/pen/BweJOv?editors=1011

**Comment by bindrid** [comment]
In all honesty, I rather attach a callback to your event handler then attach my own. Then I know you are done with the number and I don't have to worry about timing issues.

**Comment by AlexandreBonneau** [comment]
@bindrid I've created a [issue](https://github.com/autoNumeric/autoNumeric/issues/550) to track this problem.
Feel free to subscribe to it if you want to follow its resolution.

**Comment by ducman** [comment]
TLDR: Fixed by updaing your version. But event firing twice still an issue.

To clear things up on the original issue...

@VoimiX spoke about listening to the event at the `document` level via jquery. While the html5 codepen example was about listening to the event at the `element` level. **This is two different things.** As a matter of fact, you can listen to the event at the `element` level via jquery fine also. The issue is that the event wasn't propagating/bubbling up the dom.

Good news, I came upon the same issue also and found out that version 4.1.0 and up fixes this problem. You can use jquery to listen to the change event at the `document` level. @VoimiX and I were using version 4.0.x which had this problem.

Bad news, the double events is still there. I'll post what I found in #550.
